main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
```

Comment:
Do you want the user to play 8 straight games? Would it better for them to opt in to playing again?